shark_turbine/aot/fx_programs.py

```diff
@@ -167,13 +167,15 @@ def export_program(
         args=None,
         kwargs=None,
```

Comment:
Might as well type it: `strict: bool = True`

Reply:
Done.